feat(clients/vue): create deployment documentation (#1110)
jellyfin/jellyfin.org: docs/general/testing/web/index.md

:::caution
Using a development version of clients may lead to data corruption or loss on the server. Please use a dedicated test server and make regular backups of [Jellyfin files](/docs/general/administration/configuration#server-paths).
:::

:::danger

> Although CloudFlare Pages deployments from Pull Requests are built in our CI/CD workflow

Reviewer: ```suggestion
Although Cloudflare Pages deployments from Pull Requests are built in our CI/CD workflow
```

Author: Good catch 🚀

> Although Cloudflare Pages deployments from Pull Requests are built in our CI/CD workflow

Reviewer 2: ```suggestion
Although Cloudflare Pages deployments from Pull Requests are built into our CI/CD workflow
```

in a completely transparent and auditable manner, **some of them may come from external contributors that might not be good actors!**.

> By default, never trust any artifacts given by anyone outside the official channels if you can't inspect the source code first.
> They might compromise or trace your activity!

Reviewer 3: ```suggestion
By default, never trust any artifacts given by anyone outside the official channels if you without inspecting the source code first. They might compromise or trace your activity!
```

> They might compromise or trace your activity!

Reviewer 4: ```suggestion
They might compromise your system or track your activity!
```
